mkdocs.yml

```
@@ -46,7 +47,7 @@ nav:
       - CIS Benchmark:                    cis_benchmark.md
   - FAQ: FAQ.md
   - Troubleshooting: troubleshooting.md
-  - Contributing:
+  - Contribute:
```

I would leave this one as it was

```suggestion
  - Contributing:
```

kind of same as `Troubleshooting` above.